README.md

```diff
@@ -31,6 +31,14 @@ SwiftLint.lint()
 
 That will lint the created and modified files
 
+#### Inline mode
+
+If you want the lint result shows in diff instead of comment, you can use inline_mode option. Violations that out of the diff will show in danger's fail or warn section.
```

Comment:
Is there any Danger docs or code we could link to from here, to give more context about what inline mode is?

Reply:
Yeah, good idea. I will add a link to the release notes of Danger JS, it's really well written 👍